passwd.c

```diff
@@ -93,5 +106,20 @@ int main(void) {
     free(password);
     free(password_confirmation);
 
-    return EXIT_SUCCESS;
+    return ret;
+}
+
+int main(int argc, char **argv) {
+    switch (argc) {
+    case 1:
+        return main_set_password();
+    case 2:
+        if (strcmp(argv[1], "-l") == 0) {
```

Comment:
If an arg is given, and it is not `-l`, then switch will fall through, and reach end of main() function. Please print help message also if argv[1] is not `-l`

Reply:
Wouldn't the switch fall through into the `default` case, printing the help message?

Reply:
Yeah, you are right!